### PR TITLE
Await asPrometheus to follow the changes in prom-client api which now returns a promise

### PR DESCRIPTION
**Pull request by KillianG**
Issue: BB-370

### CODE REVIEW

**Review by naren-rajendran — APPROVED** [review]

File: lib/api/BackbeatAPI.js [hunk 1]
```diff
@@ -378,8 +378,8 @@ class BackbeatAPI {
      * @param {Function} cb = callback(error, data)
      * @returns {undefined}
      */
-    monitoringHandler(details, cb) {
-        const promMetrics = ZenkoMetrics.asPrometheus();
+    async monitoringHandler(details, cb) {
+        const promMetrics = await ZenkoMetrics.asPrometheus();
```
Comment (francoisferrand) [inline lib/api/BackbeatAPI.js:382]: could have been written using promise's callback api: `ZenkoMetrics.asPrometheus().then(...)`, it looks kind of contradictory to have an async method being provided with a completion callback...

**Comment by francoisferrand** [comment]
LGTM, but need rebase to "merge" the fixup commit (and then bert-e's `/reset` and re-creating the missing commits on other branches)

**Review by francoisferrand — APPROVED** [review]

**Pull request merged**